pyproject.toml

```diff
@@ -4,5 +4,12 @@ requires = [
   "setuptools_scm >= 1.15.0",
   "setuptools_scm_git_archive >= 1.0",
   "wheel",
+
+  # needed because setuptools' attr import
+  # machinery (`importlib.import_module`)
+  # imports `__init__` first
+  "ansible",
```

Comment:
This could be a real problem as it would require installing ansible twice when installing from source. Why is ansible really needed for packaging ansible-lint?

Reply:
Yes, because during the import of the version generation function it imports `__init__.py` which triggers `ImportError` otherwise.
Double install isn't a problem because of an isolated env.